src/flash/com/longtailvideo/jwplayer/controller/Controller.as

```diff
@@ -162,9 +162,11 @@ public class Controller extends GlobalEventDispatcher {
     }
 
     public function mute(muted:Boolean):Boolean {
-        if (muted !== _model.mute) {
-            _model.mute = muted;
-            return true;
+        if (_model.media) {
+            if (muted !== _model.mute) {
+                _model.mute = muted;
+                return true;
+            }
```

Comment:
I agree, this is here to match what is currently being done for volume, we should follow up on this in the future.